src/libpanic_abort/lib.rs

```diff
@@ -115,6 +115,16 @@ pub mod personalities {
     #[cfg(all(bootstrap, target_os = "windows", target_env = "gnu"))]
     pub extern "C" fn rust_eh_unwind_resume() {}
 
+    // Similar to above, this corresponds to the `eh_catch_typeinfo` lang item
+    // that's only used on Emscripten currently.
+    //
+    // Since panics don't generate exceptions and foreign exceptions are UB
+    // with -C panic=abort, any catch_unwind calls will never use this typeinfo.
```

Comment:
Could we add a note here that foreign exceptions being UB under `-Cpanic=abort` is not yet determined, right? Specifically, it's an open question exactly which ffi-unwind proposal we end up with.